### DISCUSSION TITLE
cannot install ruby 2.7.6 on macbook M1 Pro Ventura 13.2.1

### DISCUSSION THREAD

**Discussion by ghost** [rbenv/ruby-build#2158] [Build failures]
```
BUILD FAILED (macOS 13.2.1 using ruby-build 20230306)

Inspect or clean up the working tree at /var/folders/ww/r7m0gkzs5zb6zky831rpt9440000gp/T/ruby-build.20230306182919.64252.BouTIz
Results logged to /var/folders/ww/r7m0gkzs5zb6zky831rpt9440000gp/T/ruby-build.20230306182919.64252.log

Last 10 log lines:
CC="clang" /usr/bin/perl crypto/rc4/asm/rc4-md5-x86_64.pl macosx crypto/rc4/rc4-md5-x86_64.s
make[1]: *** [crypto/rc2/rc2cfb64.o] Killed: 9
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [crypto/rc2/rc2ofb64.o] Killed: 9
make[1]: *** [crypto/rand/randfile.o] Killed: 9
make[1]: *** Deleting file `crypto/rand/randfile.o'
make[1]: *** [crypto/rand/rand_unix.o] Killed: 9
make[1]: *** Deleting file `crypto/rand/rand_unix.o'
make[1]: *** [crypto/rc2/rc2_ecb.o] Killed: 9
make: *** [all] Error 2
```

I have tried a series of commands including: rbenv install 2.7.6, CFLAGS="-Wno-error=implicit-function-declaration" RUBY_CONFIGURE_OPTS='--with-readline-dir=/usr/local/opt/readline/' arch -x86_64 rbenv install 2.7.6. 

I've tried uninstalling and reinstalling rbenv using homebrew. I just keep getting these same 10 lines of errors. Even tried other versions and it just fails. The error messages don't seem to tell me anything. Any help?

**Comment by alimertozdemir** [comment]
Any solution?

**Comment by mislav** [comment]
> The error messages don't seem to tell me anything. Any help?

@AlexanderSherrow There is a line of output that starts with `Results logged to` and that tells you the location of the full build log on disk.

**Comment by DennisJMurphy** [comment]
I had a similar issue and running `Xcode-select --install` seems to have fixed it.
